pkg/prometheus/testdata/AdditionalScrapeConfigs_one_prometheus_shard.golden

```diff
@@ -20,3 +20,22 @@ scrape_configs:
     source_labels:
     - __meta_gce_label_app
     regex: my_app
+- job_name: gce_app_bar_custom_shard_relabeling
```

Comment:
Maybe I should use a custom testdata file to not fill the non-sharding goldens?

Reply:
This is fine for me